docs/source-pytorch/conf.py

```diff
@@ -412,3 +412,11 @@ def package_list_from_file(file):
 
 # ignore all links in any CHANGELOG file
 linkcheck_exclude_documents = [r"^(.*\/)*CHANGELOG.*$"]
+
+# ignore the following relative links
```

Comment:
this comment should describe not the what, but the why. Why do we need to ignore these?

Reply:
Updated this with the why.